### PR TITLE
[A.Zhukevych]CSS_practice_task1

### PR DESCRIPTION
**Pull request by Hollister009**

### CODE REVIEW

**Comment by Hollister009** [comment]
https://hollister009.github.io/ui-internship-lwo2/css-practice/task1/index.html

File: css-practice/task1/img/Shape_1.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="23" height="37" viewBox="0 0 23 37"><defs><path id="pf4ja" d="M963.154 931.416a1.66 1.66 0 0 1-1.657 1.659 1.66 1.66 0 0 1 0-3.318 1.66 1.66 0 0 1 1.657 1.66zm-.703-11.423c5.932.479 10.545 5.451 10.545 11.423v14.074c0 6.347-5.158 11.511-11.5 11.511-6.34 0-11.497-5.164-11.497-11.51v-14.075c0-5.976 4.613-10.948 10.544-11.423zm8.637 25.497v-14.074c0-5.295-4.303-9.601-9.591-9.601-5.289 0-9.59 4.306-9.59 9.6v14.075c0 5.295 4.302 9.601 9.59 9.601s9.59-4.306 9.59-9.601z"/></defs><g><g transform="translate(-950 -920)"><use fill="#fff" xlink:href="#pf4ja"/></g></g></svg>
```
Comment (tarasbobak) [inline css-practice/task1/img/Shape_1.svg]: Couldn't you think of a better name, than Shape 1, 2, 3. Name files according to what they represent

File: css-practice/task1/index.html [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Design Lab</title>
+  <link href="https://fonts.googleapis.com/css?family=Lato:300,400,700" rel="stylesheet">
+  <link rel="stylesheet" href="css/main.css">
+</head>
+
+<body>
+  <header id="hero">
```
Comment (tarasbobak) [inline css-practice/task1/index.html]: classes should be preferred over ids. This is for any your usage of id attribute.

File: css-practice/task1/css/main.css [hunk 3]
```diff
@@ -0,0 +1,512 @@
...
+}
+
+.message_white::before,
+.message_white::after {
+  background: white;
+}
+
+/* Hero Section */
+
+#hero {
```
Comment (tarasbobak) [inline css-practice/task1/css/main.css]: Id's are bad

File: css-practice/task1/css/main.css [hunk 4]
```diff
@@ -0,0 +1,512 @@
...
+@media only screen and (max-width: 900px) {
+  html {
+    font-size: 8px; 
+  }
+
+  .mouse-icon {
+    margin-top: 20rem;
+  }
+
+  .item-1>.tile-name,
```
Comment (tarasbobak) [inline css-practice/task1/css/main.css]: Spacing .item-1 > .tile-name,

File: css-practice/task1/css/main.css [hunk 5]
```diff
@@ -0,0 +1,512 @@
...
+    margin: 0;
+  }
+
+  .tile-item,
+  .tile-item-md {
+    width: calc(100% - 2rem);
+    margin: .5rem;
+  }
+
+  .item-1>.tile-name,
```
Comment (tarasbobak) [inline css-practice/task1/css/main.css]: spacing

**Comment by tarasbobak** [comment]
You don't understand when to use rems.
Please read:
https://engageinteractive.co.uk/blog/em-vs-rem-vs-px
https://www.creativebloq.com/how-to/design-a-responsive-site-with-em-based-sizing
https://css-tricks.com/rem-global-em-local/

You don't have to use rems for everything. Generally rems for font size. Then when you set font size on some component it can be styled using ems relatively to component font size rem value. That's why component can be scaled independently.

Last: rems and ems should be used for styles related to the text (font-size, paddings, line-heights etc.). It is reraly used for width and heights. For width we often use %. Height is often defined by content + padding + margin.

**Comment by tarasbobak** [comment]
I see that you tried to make it responsive. Use emulator to see how it will look on Iphone X, 8. It looks broken.

I would like you to improve your code more.

But it is pretty good anyways :)

**Comment by tarasbobak** [comment]
Compared it to design - your proportions are wrong

File: css-practice/task1/css/main.css (hunk 3 above)
Comment (AlexLarusso) [reply css-practice/task1/css/main.css]: @tarasbobak Explain why )

**Comment by tarasbobak** [comment]
Good work @Hollister009! 👍

**Pull request merged**